app/views/users/show.html.erb

```diff
@@ -1 +1,20 @@
-<h1><%= @user.name %></h1>
\ No newline at end of file
+<body>
+  <%= render "partials/user_container", user: @user %>
+        <article>
+            <h3>Bio</h3>
+            <p><%= @user.bio %></p>
+        </article>
+
+        <article>
+            <% @recent_posts.each do |post| %>
+            <%= link_to "Post ##{post.id}", user_post_path(user_id: post.author.id, id: post.id) , class: "post-heading" %>
+            <p><%= truncate(post.text, length: 110, omission: " . . .") %></p>
+            <div class="like-and-comment">
+              <p>Comments: <%= post.comment_counter || 0 %>, Likes: <%= post.like_counter || 0 %></p>
+            </div>
+              <% end %>
+        </article>
+        <%= link_to "/users/#{@user.id}/posts" do %>
+    <button>See all posts</button>
+    <% end %>
+</body>
```

Comment:
- [ ] there is an error when I load user show page, kindly change `comment_counter` to `comments_counter` and also same for `like_counter` to `likes_counter`
![Screenshot from 2023-04-21 17-09-14](https://user-images.githubusercontent.com/66097371/233684465-c709badb-3274-4a85-a6b0-8b24ade771c1.png)
- [ ] also your design is not matching the wireframe, kindly follow design
your design :point_down: 
![Screenshot from 2023-04-21 17-12-12](https://user-images.githubusercontent.com/66097371/233684816-30b83858-e19d-4d41-a141-a42e36ab0a1e.png)